doc/sphinx/source/utils/RTW/add_a_recipe.rst

```diff
@@ -0,0 +1,112 @@
+How to add a recipe to the |RTW|
+================================
+
+.. include:: common.txt
+
+.. note::
+   Before you follow these steps to add your recipe, you must be able to
+   successfully run the recipe with the latest version of ESMValTool on the
+   compute server you use at your site, as detailed by the ``platform`` option
+   in the ``[[COMPUTE]]`` section in the site-specific ``.cylc`` file in the
+   ``esmvaltool/utils/recipe_test_workflow/site/`` directory.
+
+#. Open a `new ESMValTool issue`_ on GitHub, assign yourself to the issue, and
+   add the ``Recipe Test Workflow (RTW)`` label to the issue, see
+   `ESMValTool issue #3663`_ for an example.
+
+#. Create a branch.
+
+#. Obtain the duration and memory usage of the recipe from the messages printed
+   to screen, or at the end of the ``run/main_log.txt`` file in the recipe
+   output directory after running your recipe on the compute cluster you use at
+   your site; these messages will look something like::
+
+     YYYY-MM-DD HH:MM:SS:sss UTC [12345] INFO    Time for running the recipe was: 0:02:13.334742
+     YYYY-MM-DD HH:MM:SS:sss UTC [12345] INFO    Maximum memory used (estimate): 2.4 GB
+     [...]
+     YYYY-MM-DD HH:MM:SS:sss UTC [12345] INFO    Run was successful
+
+#. Add the recipe to the ``[task parameters]`` section in the
+   ``esmvaltool/utils/recipe_test_workflow/flow.cylc`` file.
+
+   .. hint::
+      If the recipe takes less than 10 minutes to run then it should be added
+      to the ``fast`` option. Recipes that take longer than ten minutes should
+      be added to the ``medium`` option.
```

Comment:
Are these `fast` and `medium` options something common in all platforms?

Reply:
At the moment, yes. But I realised when trying to add recipes for DKRZ that they would fail at other sites (since the resources and KGOs are required for each site). So there may be different `fast` and `medium` recipe lists in the future.